src/modules/dashboards/utils/createCodeSnippet.test.ts

```diff
@@ -25,19 +27,35 @@ test('return code snippet for provided projectId and dashboardId', () => {
       </head>
       <body>
         <div id=\\"root\\"></div>
+        <div id=\\"modal-root\\"></div>
       <script type=\\"text/javascript\\">
-         new KeenPublicDashboard({
-           container: '#root',
-           dashboardId: 'dashboardId',
-           backend: {
-             analyticsApiUrl: 'api.keen.io',
-             dashboardsApiUrl: 'https://blob-service.keen.io',
-           },
-           project: {
-             id: 'projectId',
-             accessKey: 'userKey' 
-           },
-         }).render();
+        new KeenPublicDashboard({
+          container: '#root',
+          modalContainer: '#modal-root',
+          dashboardId: 'dashboardId',
+          defaultTimezoneForQuery: \\"Etc/UTC\\",
+          disableTimezoneSelection: false,
+          widgetsConfiguration: {
+            datePicker: {
+              defaultTimezone: 'Etc/UTC',
+              disableTimezoneSelection: false
+            }
+          },
+          backend: {
+            analyticsApiUrl: 'api.keen.io',
```

Comment:
The backend is obsolete here I believe.

Reply:
<img width="1402" alt="Screenshot 2021-04-30 at 08 46 56" src="https://user-images.githubusercontent.com/23423731/116658704-ae531f80-a990-11eb-8ad8-5f0cc4482057.png">

without analyticsApiUrl we're unable to get results

Reply:
Ok, I wasn't aware of it. I believe we need to set default value for `analyticsApiUrl` to `api.keen.io` then.